### ISSUE TITLE
Confusing Blocks in Groups Documentation Code snippet

### ISSUE THREAD

**Issue by ajibigad** [gin-gonic/gin#3059]
Hi, 

This is a question or suggestion.

The blocks used in this [example](https://github.com/gin-gonic/gin#grouping-routes) of grouping routes are quite confusing. 

I had similar confusion as asked in this [question](https://stackoverflow.com/questions/53936092/syntax-question-from-gin-gonic-documentation).

At a glance, it looked like the blocks were a part of the `router.Group("/v1")` statement, like a struct being initialized with a struct literal.

My question: Are there any reasons the blocks were used? 

If there aren't, I think the blocks should be removed to avoid confusion.

Thanks

**Comment by SJrX** [comment]
Just saw this bug report, I ended up hitting up the Liberia.chat go channel with this a few weeks ago being very confused about what that was actually doing if anything.

**Comment by miles-ton** [comment]
yes, i agree with this is very confusing. when i first saw this code, i thought this block was the part of Group() returned value initializing code. after analysis, i think it just a simply separating code block, have not any other semantic.
